AllJoyn/Platform/BridgeHost/App.xaml.cs

```diff
@@ -0,0 +1,359 @@
+﻿//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+
+using System;
+using Windows.ApplicationModel.Activation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+using Windows.ApplicationModel.Background;
+using Windows.ApplicationModel.AppService;
+using Windows.Foundation.Collections;
+using System.Linq;
+using BridgeRT;
+using Windows.Storage;
+using Windows.ApplicationModel;
+using System.Text;
+using Windows.ApplicationModel.DataTransfer;
+using System.IO;
+using System.Runtime.Serialization;
+using System.Runtime.Serialization.Json;
+using System.Threading.Tasks;
+using System.Diagnostics;
+
+// The Blank Application template is documented at http://go.microsoft.com/fwlink/?LinkId=402347&clcid=0x409
+
+namespace OpenT2T.Bridge
+{
+    [DataContract]
+    internal class DeviceProps
```

Comment:
Now I have changed it to plumb props as-is. But app service looks for 'id, which is the only must-be-present value, since that is used to uniquely identify the device. THis is because it stores the information about the device and is used to reject multiple onboards of same device  (either requested by the app or when it onboards automatically on start up)

---
In reply to: [75555046](https://github.com/tjaffri/msiot-samples/pull/9#discussion_r75555046) [](ancestors = 75555046,75537421,75206432,75187292)